eZ/Publish/API/Repository/Values/Content/Query/Criterion/UserEnabled.php

```diff
@@ -0,0 +1,30 @@
+<?php
+declare(strict_types=1);
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Publish\API\Repository\Values\Content\Query\Criterion;
+
+use eZ\Publish\API\Repository\Values\Content\Query\Criterion;
+use eZ\Publish\API\Repository\Values\Content\Query\Criterion\Operator\Specifications;
+
+class UserEnabled extends Criterion
```

Comment:
Please keep the existing convention for the boolean criterion e.g. IsMainLocation, IsFieldEmpty etc.. 

```suggestion
class IsUserEnabled extends Criterion
```